src/twilio/twilio_endpoint.bal

```diff
@@ -26,10 +26,10 @@ import ballerina/mime;
 # + authyClient - HTTP client endpoint for authy api
 public type Client client object {
 
-    public string accountSId;
-    public string xAuthyKey;
-    public http:Client basicClient;
-    public http:Client authyClient;
+    public string accountSId = "";
```

Comment:
Let's use nillable and add validate in the init.

Reply:
Fixed in https://github.com/wso2-ballerina/module-twilio/pull/56/commits/d683736e857c7ee182a9c65e1974506da6ecc0a5. Actually we don't need to add validation step to the init function to check `accountSId ` and  `xAuthyKey`, because those attributes are mandatory to initialize a `TwilioConfiguration` object.